components/admin_console/team_channel_settings/channel/details/channel_details.test.tsx

```diff
@@ -4,19 +4,22 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 
-import ChannelDetails from './channel_details.jsx';
+import {Group} from 'mattermost-redux/types/groups';
+import {Channel} from 'mattermost-redux/types/channels';
+
+import ChannelDetails from './channel_details';
 
 describe('admin_console/team_channel_settings/channel/ChannelDetails', () => {
     test('should match snapshot', () => {
-        const groups = [{
+        const groups: Partial<Group>[] = [{
```

Comment:
Type 'Partial<Group>[]' is not assignable to type 'Group[]'.ts(2322) . on line 33